src/components/override-sdk/field/Checkbox/Checkbox.tsx

```diff
@@ -56,7 +59,27 @@ export default function CheckboxComponent(props) {
   return (
     <>    
       {exclusiveOption && <div className="govuk-checkboxes__divider">or</div>}
-      <GDSCheckbox
+
+      {/* If its the declaration view then group the checkboxes separately so the error message is assigned correctly */}
+      {OverrideLabelValue === 'Declaration' ? (
```

Comment:
need to consider the localised label value as well for welsh

Reply:
Make sure this code works for welsh as well

Reply:
<img width="608" alt="Screenshot 2023-11-17 131151" src="https://github.com/norm-l/odx/assets/149371097/47e3cc43-4a41-436e-b53e-8bd50bef8021">

Reply:
The functionality works for both English and Welsh. I've also added `trim()` and `toLowerCase()` just to make sure the text is always the same on the `OverrideLabelValue`.